src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -83,6 +89,16 @@
                 <StackPanel>
                     <CheckBox x:Name="Rename_asynchronously_exerimental"
                               Content="{x:Static local:AdvancedOptionPageStrings.Option_Rename_asynchronously_experimental}" />
+
+                    <Label x:Name="Rename_UI_setting_label" Content="{x:Static local:AdvancedOptionPageStrings.Where_should_the_rename_ui_be_shown}" />
```

Comment:
a bit odd that `ui` is lowercased.

Reply:
maybe just `Rename textbox location: inline/in-top-right`?

Reply:
That's not completely accurate though. It's a bit more nuanced than that. At most, inline rename options or dashboard them?